runelite-client/src/main/java/net/runelite/client/plugins/chathistory/ChatHistoryPlugin.java

```diff
@@ -254,7 +254,8 @@ public void onMenuOptionClicked(MenuOptionClicked event)
 	{
 		final String menuOption = event.getMenuOption();
 
-		if (CLEAR_HISTORY.equals(menuOption))
+		// The menu option for clear history is "<col=ffff00>Public:</col> Clear history"
+		if (menuOption.contains(CLEAR_HISTORY))
```

Comment:
Perhaps `menuOption.endsWith(CLEAR_HISTORY)` to be more explicit?

Reply:
I almost did this, but contains seemed "safer" if jagex changes this. I'll use `.endsWith` tho. It's a quick fix anyway.